### ISSUE TITLE
File permission problem on xfs

### ISSUE THREAD

**Issue by willseward** [docker/docker-bench-security#117]
Shouldn't this pass 3.26?
```
srw-rw----. 1 root docker 0 Feb 16 15:26 /var/run/docker.sock
```

**Comment by konstruktoid** [comment]
Hi @willseward, thanks for reporting this. I'll have a look.

**Comment by konstruktoid** [comment]
It should pass 3.26 but the script didn't handle /var/run/docker.sock with an SELinux security context.
Please test and comment on https://github.com/docker/docker-bench-security/pull/118.

**Comment by konstruktoid** [comment]
Closed by #118
